### PR TITLE
[WIP] Adding weekly cron for cluster-density, cluster-density-v2 and node-density on Rosa

### PR DESCRIPTION
**Pull request by svetsa-rh**
Add new chain and jobs to run cluster-density, node-density and network-perf on Rosa

### CODE REVIEW

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-ocp-qe-perfscale-ci-tests-rosa-all

**Comment by svetsa-rh** [comment]
/pj-rehearse  pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-images periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-ocp-qe-perfscale-ci-tests-rosa-all periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse ci/rehearse/openshift-qe/ocp-qe-perfscale-ci/main/rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-ocp-qe-perfscale-ci-tests-rosa-all

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-ocp-qe-perfscale-ci-tests-rosa-all

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa

File: ci-operator/step-registry/rosa/aws/sts/deprovision/rosa-aws-sts-deprovision-chain.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 chain:
   as: rosa-aws-sts-deprovision
   steps:
+  - chain: gather
```
Comment (yasun1) [inline ci-operator/step-registry/rosa/aws/sts/deprovision/rosa-aws-sts-deprovision-chain.yaml:4]: That is the correct place to add the `must-gather` step.

**Comment by yasun1** [comment]
@svetsa-sh You can run `periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-sts-p2-f7` to verify the classic rosa cluster, and `periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-critical-p1-f2` to validate the rosa hypershift cluster.

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-nodedensity-ocp-qe-perfscale-ci-tests-rosa-nodedensity periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-sts-p2-f7 periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-critical-p1-f2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-sts-p2-f7 periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-critical-p1-f2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-sts-p2-f7 periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-critical-p1-f2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-all-ocp-qe-perfscale-ci-tests-rosa-all periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-rosa-sts-hypershift-guest-stage-critical-p1-f2 periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-rosa-sts-p2-f7 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-images periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-images periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-cluster-density-v2-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-images

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.13-nightly-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density

**Comment by svetsa-rh** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests

**Comment by svetsa-rh** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-images pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-prod-images periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-ocp-qe-perfscale-ci-tests-rosa-node-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-ocp-qe-perfscale-ci-tests periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-nightly-ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2 periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa-4.12-prod-ocp-qe-perfscale-ci-tests-rosa-hypershift-cluster-density

**Pull request closed without merging**